### ISSUE TITLE
Pipe Minigame: ReferenceError pipeminigame is not defined

### ISSUE THREAD

**Issue by Zetawilk** [krues8dr/rpgmakermv-plugins#7]
A new RPG Maker MV project was created. All default plugins were disabled. The plugins Kru_Core and Kruo_PipeMinigame were added and enabled with default settings. 

An event was created with the following contents:
Script... > `pipeminigame();`

When interacting with the event in-game via button press, the game returns a Reference Error, citing the following:
"pipeminigame is not defined"

![pipeminigame plugin manager](https://user-images.githubusercontent.com/6036009/87960139-e16f9680-ca81-11ea-9977-669483e8da54.png)
![pipeminigame plugin settings](https://user-images.githubusercontent.com/6036009/87960140-e2082d00-ca81-11ea-850c-05b2b52fd90c.png)
![pipeminigame referror](https://user-images.githubusercontent.com/6036009/87960142-e2a0c380-ca81-11ea-8c84-9033a30a110f.png)

**Comment by krusynth** [comment]
@Zetawilk Thanks for the report!  

I found the issue: RPG Maker MV uses an older engine that doesn't support modern Javascript - I've been testing mostly in a web browser.  As a result, it was dying on class properties, which aren't supported apparently. I've retrofit the plugin, please let me know if this doesn't solve the problem! 

I've also added new options to allow you to choose your images for the pipe tiles; you can  [download a copy of my demo image from here](https://raw.githubusercontent.com/wiki/krusynth/rpgmakermv-plugins/img/Kru_PipeMinigame/pipetiles.png) and modify it to fit your needs.

**Comment by Zetawilk** [comment]
@krusynth The plugin now works for me! Thank you very much. 

I have tested the plugin both in a fresh project with no other plugins, as well as in a project that contains a slew of various other plugins, mostly including plugins from Galv, Khas, MrTS, Yanfly, and various others, including other plugins listed here. 

In the fresh project, the game worked as expected, although scrolling up and down in the list of pipe pieces seemed to continue in either direction without looping around for an indeterminate length. 

In the project with many plugins, I had to place the Core and PipeMinigame plugins above the YEP (Yanfly) plugins for the game to display properly, and navigating between pieces looped at the top and bottom. While I'm satisfied, I also tested out other options that I thought some might question. When tested with more than five pieces available, the tile selection window did not scroll. While selecting the tiles themselves with the mouse works properly, there's no method to rotate them with the mouse, and it did not seem possible to select where to place the tiles using the mouse, or to place them down using the mouse. I just thought that some users might question these aspects, so I mentioned them here. 

Thanks for your work. I greatly enjoy the Pipe Minigame plugin!
